web_src/js/features/repo-legacy.js

```diff
@@ -86,11 +86,12 @@ export function initRepoCommentForm() {
     let hasUpdateAction = $listMenu.data('action') === 'update';
     const items = {};
 
-    $(`.${selector}`).dropdown('setting', 'onHide', () => {
-      hasUpdateAction = $listMenu.data('action') === 'update'; // Update the var
-      if (hasUpdateAction) {
-        // TODO: Add batch functionality and make this 1 network request.
-        (async function() {
+    $(`.${selector}`).dropdown({
+      'action': 'nothing', // do not hide the menu if user presses Enter
```

Comment:
This is needed to make search work well, without it a label `color/green` is not found by searching for `green`.

It's because this code from `web_src/js/features/common-global.js` no longer applies.
```
  $uiDropdowns.filter(':not(.custom)').dropdown({
    fullTextSearch: 'exact'
  });
```